test/parallel.jl

```diff
@@ -13,6 +13,12 @@ id_other = filter(x -> x != id_me, procs())[rand(1:(nprocs()-1))]
 @fetch begin myid() end
 
 d = drand((200,200), [id_me, id_other])
+dc = copy(d)
+
+@test d == dc                                           # Should be identical
+@spawnat workers()[1] localpart(dc)[1] = 0
+@test fetch(@spawnat workers()[1] localpart(d)[1] != 0) # but not point to the same memory
```

Comment:
Should be `@spawnat id_other localpart(dc)[1] = 0` and `@test fetch(@spawnat id_other localpart(d)[1] != 0)` ?